src/getting-started.md

```diff
@@ -1,10 +1,16 @@
 # Getting started
 
-This section is designed to help you integrate the FIB API seamlessly into your application or system by providing instructions on how to register an account, check the API availability, call API endpoints, and test your implementation in a sandbox environment.
+Since FIB allows its users to interact with the network both via API and the Web App, this section provides instructions for all necessary preliminary steps, be you a regular user or a developer aiming to integrate the FIB API into their application or system.
 
-It is advised to read the following topics in the order that they are presented:
+Regular users should start with the [Registering an account](getting-started/registering-an-account.md) tutorial, and then move on to the [Tutorials: Web App](tutorials-web.md) section that explains how to interact with the FIB network via [FIB Web App](overview/web-interface.md).
```

Comment:
```suggestion
Users should start with the [Registering an account](getting-started/registering-an-account.md) tutorial, and then move on to the [Tutorials: Web App](tutorials-web.md) section that explains how to interact with the FIB network via [FIB Web App](overview/web-interface.md).
```

For anyone it's crucial to understand the flows imo

Reply:
Moreover registering and navigating to the profile page is required even for solely API integrator